doc/api/configuration.md

```diff
@@ -256,15 +256,21 @@ res = %sql SELECT * FROM languages
 
 ## `named_parameters`
 
-```{versionadded} 0.9
+```{versionchanged} 0.10.9
 ```
 
-Default: `False`
+Default: `warn`
```

Comment:
add a note on the possible side-effects when using `disabled`,

you can add an important [admonition](https://jupyterbook.org/en/stable/reference/cheatsheet.html#admonitions), mention that this might have some side effects due to the use of sqlalchemy (explain it with some detail) and mention that people can open an issue on slack if they encounter problems